src/config/Logos.ts

```diff
@@ -5,3 +5,10 @@ export const IstioLogo = require('../assets/img/istio-logo.svg');
 
 /** Kiali logo */
 export const KialiLogo = require('../assets/img/logo-alt.svg');
+
+// Runtimes
+export const runtimes = {
+  'Vert.x': 'vertx-logo.png',
+  'Node.js': 'nodejs-logo.png',
+  'Thorntail': 'thorntail-logo.png'
+};
```

Comment:
I don't know if create the full path here instead that in the code, all the images should be in the same place.

WorkloadDescription.tsx L-28
```ts
      const logo = require('../../../assets/img/' + imageName);
      return <img src={logo} alt={name} />;
```
we could do something like 
```ts
const imgPath = '../assets/img/');
/** Kiali logo */
export const KialiLogo = require(imgPath + 'logo-alt.svg');

// Runtimes
export const runtimes = {
  'Vert.x': imgPath + 'vertx-logo.png',
  'Node.js': imgPath +'nodejs-logo.png',
  'Thorntail': imgPath + 'thorntail-logo.png'
};
```
No blocker . What do you think @jotak ?

Reply:
mm I think that this not fix that because is different path but you cant add the require and use it like we do with KialiLogo and remove the require of the WorkloadDescription code

Reply:
Yes, good point, I changed it. However I can't assign `imgPath` and then build the string by concatenating, I get a "module not found" error, I guess this is because `require` calls are preprocessed by webpack and not executed at runtime (?). Setting full relative path as a single string works fine however.